SPECS/xen.spec

```diff
@@ -25,7 +25,7 @@
 Summary: Xen is a virtual machine monitor
 Name:    xen
 Version: 4.7.5
-Release: 5.8.1.xcp
+Release: 5.9.1.xcp
```

Comment:
Either 5.8.2.xcp or 5.9.xcp : 5.8 is the release from XS, and .2 is the revision from XCP-ng 
... or 5.9.xcp since we're not expecting any 5.9 release from XS so we could choose to create our own.

Let's make it 5.9.xcp

Reply:
Done